README.md

```diff
@@ -43,10 +43,15 @@ Quoted strings are terminated by a matching quote at the end of a line
 to continue a quoted string, prefix it with a `\`.
 
 The m(enu) command uses `,` as a delimiter between label and command and `|` as a delimiter
-between entries (label+command). If you want to use these 2 characters in a label or command, you have to escape 
-them with `\`.
-
-Example command: `echo "m Browser,firefox|Terminal,xterm" > /tmp/test` (where `mkfifo /tmp/test` has been executed before)
+between entries (label+command).   
```

Comment:
I think this reads better if we don't have these additional line-breaks. Just keep it all as one paragraph with regular text-wrap linebreaks.